unitsrouter.cpp:20:61: warning: unused parameter 'locale' [-Wunused-parameter]
UnitsRouter::UnitsRouter(MeasureUnit inputUnit, StringPiece locale, StringPiece usage,
                                                            ^
unitsdata.cpp:19:1: error: version control conflict marker in file
<<<<<<< HEAD
^
unitsdata.cpp:103:17: error: use of undeclared identifier 'trimSpaces'
                trimSpaces(cr->factor, status);
                ^
unitsdata.cpp:248:83: warning: unused parameter 'inputUnit' [-Wunused-parameter]
void getUnitsData(const char *outputRegion, const char *usage, const MeasureUnit &inputUnit,
                                                                                  ^
1 warning and 2 errors generated.

--- a/icu4c/source/i18n/unitsdata.cpp
+++ b/icu4c/source/i18n/unitsdata.cpp
@@ -16,6 +16,8 @@ U_NAMESPACE_BEGIN
 
 namespace {
 
+using icu::number::impl::DecimalQuantity;
+
 void trimSpaces(CharString& factor, UErrorCode& status){
    CharString trimmed;
    for (int i = 0 ; i < factor.length(); i++) {

--- a/icu4c/source/i18n/unitsdata.cpp
+++ b/icu4c/source/i18n/unitsdata.cpp
@@ -16,9 +16,6 @@ U_NAMESPACE_BEGIN
 
 namespace {
 
-<<<<<<< HEAD
-using icu::number::impl::DecimalQuantity;
-=======
 void trimSpaces(CharString& factor, UErrorCode& status){
    CharString trimmed;
    for (int i = 0 ; i < factor.length(); i++) {
@@ -29,7 +26,6 @@ void trimSpaces(CharString& factor, UErrorCode& status){
 
    factor = std::move(trimmed);
 }
->>>>>>> 9bcc4b698ff4b2afbf321188bceff809a27342f2
 
 /**
  * A ResourceSink that collects conversion rate information.

